src/colorize.cr

```diff
@@ -125,8 +125,9 @@ module Colorize
   class_property? enabled : Bool = true
 
   # Makes `Colorize.enabled` `true` if and only if both of `STDOUT.tty?` and `STDERR.tty?` are `true`.
+  # And the terminal is not a dumb terminal that will not understand escape sequences.
```

Comment:
This doesn't make sense as a standalone sentence.

Reply:
I have changed `Spec` to use `on_tty_only!`